spark/common/src/main/scala/org/apache/sedona/sql/UDF/AbstractCatalog.scala

```diff
@@ -86,6 +87,10 @@ abstract class AbstractCatalog {
         functionIdentifier,
         expressionInfo,
         functionBuilder)
+      FunctionRegistry.builtin.registerFunction(
```

Comment:
Can you confirm if this gets each function registered twice? (once for temp function, one for built-in)?

How about `aggregateExpressions?`